lib/configuration_selector.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'board'
+require_relative 'game_builder'
+require_relative 'player_selector'
+require_relative 'order_selector'
+require_relative 'players_builder'
+
+class ConfigurationSelector
+  def initialize(console)
+    @console = console
+  end
+
+  def select_options
+    @console.player_menu
+    player_names = choose_players
+
+    @console.order_menu(player_names)
+    ordered_players = choose_order(player_names)
+
+    board = Board.new
```

Comment:
you should initialized board in `GameBuilder`. we dont need to pass any params around right now but leaving the details to the builder is one of the perks of the pattern. `builder.set_board`